.github/workflows/ci.yml

```
@@ -0,0 +1,29 @@
+name: ci
+
+on:
+  push:
+    branches:
+      - main
+
+defaults:
+  run:
+    shell: bash --noprofile --norc -euo pipefail {0}
+
+env:
+  # Use Maven wrapper from repo with Maven version and other configs
+  MAVEN: ./mvnw -B
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v3
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'zulu'
```

```suggestion
          distribution: 'temurin'
```
:art: Wondering if it should be consistent to the JDK used in Trino.

---

we are .. Trino uses Zulu and suggests that in the docs

---

ah! yes you are completely right 👍 I guess I saw a `temurin` reference somewhere and misjudged things. ❤️

---

Actually, I think it was a recent change https://github.com/trinodb/trino/pull/17685 and docs didn't follow 🤷 I'm happy to help out let me know if this is smth that needs cleaning.

---

Changed back to Temurin now and will update the Trino docs.